terraform/modules/env-metrics/azul.tf

```diff
@@ -0,0 +1,108 @@
+locals {
```

Comment:
Please remove this file from this PR. The health check itself is sufficient to report on the health status of the API. If you are using APIGateway, you can obtain general HTTP metrics (including status classes) from CloudWatch metrics. See

https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#metricsV2:graph=~();namespace=AWS/ApiGateway;dimensions=ApiName